lib/src/theme/macos_colors.dart

```diff
@@ -17,50 +17,156 @@ class MacosColors {
+
+  static const systemBrownColor = MacosColor(0xffA2845E);
+  static const systemBrownColorDark = MacosColor(0xffAC8E68);
+
+  static const systemPinkColor = MacosColor(0xffFF2D55);
+  static const systemPinkColorDark = MacosColor(0xffFF375F);
+
+  static const systemPurpleColor = MacosColor(0xffAF52DE);
+  static const systemPurpleColorDark = MacosColor(0xffBF5AF2);
+
+  static const systemTealColor = MacosColor(0xff55BEF0);
+  static const systemTealColorDark = MacosColor(0xff5AC8F5);
+
+  static const systemIndigoColor = MacosColor(0xff5856D6);
+  static const systemIndigoColorDark = MacosColor(0xff5E5CE6);
+
+  static const systemGrayColor = MacosColor(0xff8E8E93);
+  static const systemGrayColorDark = MacosColor(0xff98989D);
+
+  /// A link to other content.
+  static const linkColor = MacosColor(0xff0068DA);
+  static const linkColorDark = Color.fromRGBO(65, 156, 255, 1);
+
+  /// A placeholder string in a control or text view.
   static const placeholderTextColor = MacosColor(0xff737473);
   static const windowFrameColor = MacosColor(0xffddddde);
+
+  /// The text of a selected menu.
   static const selectedMenuItemTextColor = MacosColor(0xfffeffff);
+
+  /// The text on a selected surface in a list or table.
   static const alternateSelectedControlTextColor = MacosColors.white;
+
+  ///	The text of a header cell in a table.
   static const headerTextColor = MacosColors.white;
+
+  ///	A separator between different sections of content.
   static const separatorColor = MacosColor(0xff39373c);
+
+  ///	The gridlines of an interface element such as a table.
   static const gridColor = MacosColor(0xff39373c);
+
+  ///	The text in a document.
   static const textColor = MacosColors.white;
+
+  ///	Text background.
   static const textBackgroundColor = MacosColor(0xff1e1e1e);
+
+  ///	Selected text.
   static const selectedTextColor = MacosColors.white;
+
+  ///	The background of selected text.
   static const selectedTextBackgroundColor = MacosColor(0xff3f638b);
-  static const unemphasizedSelectedTextBackgroundColor = MacosColor(0xff464646);
+
+  /// A background for selected text in a non-key window or view.
+  static const unemphasizedSelectedTextBackgroundColor = MacosColor(0xffDCDCDC);
+  static const unemphasizedSelectedTextBackgroundColorDark =
+      MacosColor(0xff464646);
+
+  /// Selected text in a non-key window or view.
   static const unemphasizedSelectedTextColor = MacosColors.white;
+
+  /// The background of a window.
   static const windowBackgroundColor = MacosColor(0xff3b373d);
+
+  /// The background behind a document's content.
   static const underPageBackgroundColor = MacosColor(0xff282828);
-  static const controlBackgroundColor = MacosColor(0xff1e1e1e);
+
+  /// The background of a large interface element, such as a browser or table.
+  static const controlBackgroundColor = MacosColor(0xffFFFFFF);
+  static const controlBackgroundColorDark = MacosColor(0xff1E1E1E);
```

Comment:
These could be only one:

```dart
static const Color controlBackgroundColor = CupertinoColor.fromBrightness(
  lightColor: MacosColor(0xffFFFFFF), // Could be MacosColors.white too
  darkColor: MacosColor(0xff1E1E1E),
),
```

Reply:
Tbh I don't really understand how `fromBrightness` works because we aren't specifying a `Brightness`

Reply:
Yeah I tested this out and it does not work.

Reply:
> I mean I see that it checks Platform Brightness but do we know for a fact that the correct brightness will get returned for a MacosApp?

`fromBrightness` is only to tell what color to use at the time. To get the correct color, use `MacosDynamicColor.resolve(MacosColor.controlBackgroundColor, context)`